node/Dockerfile.arm64

```diff
@@ -149,9 +149,7 @@ RUN rpm -i /tmp/rpms/iptables-libs-${IPTABLES_VER}.el8.aarch64.rpm && \
 RUN microdnf --enablerepo=baseos install iproute-tc nftables
 
 # Install iptables via rpm. Install compatible libnftnl version with the selected iptables version
-RUN rpm --force -i /tmp/rpms/libnftnl-${LIBNFTNL_VER}.el8.aarch64.rpm && \
-    # Install both and select at runtime.
-    rpm -i /tmp/rpms/iptables-legacy-${IPTABLES_VER}.el8.2.aarch64.rpm && \
+RUN rpm -i /tmp/rpms/iptables-legacy-${IPTABLES_VER}.el8.2.aarch64.rpm && \
```

Comment:
we should update the comment (and is there any step in lines 57-75 that can be removed since we're not installing this package anymore?)